### PR TITLE
Adding tests for the agents module

### PR DESCRIPTION
**Pull request by rihp**
### PR Quality Checklist
- [ ] My pull request is atomic and focuses on a single change.
- [ ] I have thoroughly tested my changes with multiple different prompts.
- [ ] I have considered potential risks and mitigations for my changes.
- [ ] I have documented my changes clearly and comprehensively.
- [ ] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Comment by ntindle** [comment]
Update this from master

File: autogpt/utils.py [hunk 1]
```diff
@@ -49,15 +49,17 @@ def readable_file_size(size, decimal_places=2):
     return f"{size:.{decimal_places}f} {unit}"
 
 
-def get_bulletin_from_web() -> str:
```
Comment (winglian) [inline autogpt/utils.py:52]: why remove the type hint here?

Comment (ntindle) [reply autogpt/utils.py:52]: It’s from a diff pr, this one needs help rebasing

Comment (winglian) [reply autogpt/utils.py:52]: yeah, it's in a very bad state atm though that rebasing won't help. There are broken things added like this https://github.com/Significant-Gravitas/Auto-GPT/commit/f4db77a7b0513effbc317d5b374fdd9ad9ecf161 where the merge conflict was committed and then it references code that doesn't exist in either upstream or that branch...

**Comment by p-i-** [comment]
This is a mass message from the AutoGPT core team.
Our apologies for the ongoing delay in processing PRs.
This is because we are re-architecting the AutoGPT core!

For more details (and for infor on joining our Discord), please refer to:
  https://github.com/Significant-Gravitas/Auto-GPT/wiki/Architecting

**Pull request closed without merging**